_source/_posts/2021-02-22-first-nestjs-application.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: blog_post
+title: "Build Your First NestJS Application"
+author: nickolas-fisher
+by: contractor
+communities: [javascript]
+description: "Learn how to build a basic application using NestJS."
+tags: [nestjs, javascript]
+tweets:
+- "Wanna learn how to build a basic application in NestJS? Check this out!"
+- "Need to get started with NestJS? We've got you covered!"
+- "Get your head around the basics of building applications with NestJS!"
+image: blog/first-nestjs-application/social-image.png
+type: conversion
+---
+
+NestJs is a popular Node.js framework that is built with typescript and makes use of object-oriented programming, functional programming, and functional reactive programming. NestJs boasts that it provides a framework for building scalable server-side applications. 
+
+NestJs integrates nicely with Okta's single sign-on provider. Okta makes securing a web service, such as the one you will build, quick and easy. 
+
+## Create your Okta Application with the CLI
+
+The first thing you will need to do is sign up for an [Okta Developer's account](https://developer.okta.com/signup/) if you haven't already. his account is free forever and allows you to start building applications secured with Okta immediately. Next, you will need to create an application in Okta. To do this you will use the [Okta CLI](https://developer.okta.com/blog/2020/12/10/introducing-okta-cli). If you haven't worked with the CLI yet you can check out the [full documentation here](https://cli.okta.com/).
```

Comment:
The okta cli creates the okta org for you. No need to go to /signup. Just use: `okta start`. Also, you wrote: 

```
his account is free...
```
(instead of This)